client/packages/dashboard/src/widgets/ReplenishmentWidget.tsx

```diff
@@ -30,6 +40,15 @@ export const ReplenishmentWidget: React.FC<PropsWithChildrenOnly> = () => {
     error: requisitionCountError,
   } = useDashboard.statistics.requisitions();
 
+  const { customDate } = useFormatDateTime();
+  const dateTimeFormat = 'yyyy-MM-dd HH:mm';
+  const today = new Date();
+  today.setHours(0, 0, 0, 0);
+  const endDay = new Date(today);
+  endDay.setHours(23, 59, 59, 999);
```

Comment:
two problems with the date:
1. it won't match anything because there won't be any inbounds with a created date in the future
2. it doesn't match what the InvoiceCount service is doing, which is taking the start date as the Monday of the current week. 

fortunately there's a helper for that!
date-fns has [previousMonday](https://date-fns.org/v2.30.0/docs/previousMonday) which is what we need!

just use previousMonday as the start and the end of today as the end date - that should give you what the API is counting

Reply:
1. As long as an invoice has been created today after 0000 and before 2359, it will match (: 
2. Ah yes, I see it now! Will fix (: thanks Mark~